cmd/root.go

```diff
@@ -162,16 +168,40 @@ func GetRootCmd() cobra.Command {
 	return *rootCmd
 }
 
-func parseArgs(args []string) []string {
+// parseArgs parses the command-line positional arguments that contain file glob patterns.
+// If no argument is provided, return the default path (current directory).
+// Perform glob pattern expansion.
+func parseArgs(args []string) ([]string, error) {
 	if len(args) == 0 {
 		args = parser.DefaultPath
 	}
 
 	if stdin {
 		args = []string{os.Stdin.Name()}
 	}
+	// Perform glob expansion.
+	var files []string
+	for _, arg := range args {
+		var f []string
+		var err error
+		if strings.Contains(arg, "**") {
```

Comment:
woke is already walking the directory tree in the parser. This seems to defeat that purpose - https://github.com/get-woke/woke/blob/main/pkg/parser/parser.go#L129